app/views/application/_user.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="col-md-12">
+	<div class="media blog-media">
+		<a href="blog-post-left-sidebar.html"><img class="d-flex" src="<%= user.photo%>"
+				alt="<%= user.photo%>" height="256" width="256"></a>
+		<div class="circle">
+			<h6 class="day">Last Post</h6>
+			<span class="month">4 sep</span>
+		</div>
+		<div class="media-body">
+			<a href="/users/<%=user.id%>">
+				<h1 class="mt-4 text-center display-4">
+					<%= user.name%>
+				</h1>
+			</a>
+			<ul>
+				<li>
+					<h3>Number of Posts: <%= user.posts_count.present? ? user.posts_count : 0 %>
+					</h3>
```

Comment:
- [ ] Please move the username to the left as to match the given wireframe: 
![image](https://github.com/Izebiafe/Blog/assets/91486791/d437a817-f649-437f-be01-4cd1c2a4db5f)
![image](https://github.com/Izebiafe/Blog/assets/91486791/209c2880-e231-435b-8b9a-211bb4951081)